CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.13.3 / 2015-05-11
+* [BUGFIX] Handle fingerprint collisions properly.
+* [CHANGE] Comments in rules file must start with "#".
```

Comment:
How about:

Comments in rules files must now start with `#`. The undocumented `//` and `/* ... */` comment styles are no longer supported.